searches.view.lkml

```diff
@@ -439,91 +414,166 @@ view: searches {
     group_label: "OS"
   }
 
+  # Device
+  # duplicates throughout this section
+
+  dimension: device_type {
+    type: string
+    sql: ${TABLE}.dvce_type ;;
+    group_label: "Device"
+  }
+
+  dimension: device_is_mobile {
+    type: yesno
+    sql: ${TABLE}.dvce_ismobile ;;
+    group_label: "Device"
+  }
+
+  # Search
+
+  # duplicate from atomic.events (events.view.lkml)
+  dimension_group: search_time {
+    type: time
+    timeframes: [hour,date,week,month,quarter,year,raw]
+    sql:  ${TABLE}.collector_tstamp ;;
+    group_label: "Search"
+  }
+
+  # search_id: string
+  # Table reference - VARCHAR(255) ENCODE ZSTD NOT NULL
+  # Unique ID for each click (equal to event_id in atomic.events)
+  dimension: search_id {
+    type: string
+    sql: ${TABLE}.search_id ;;
+    group_label: "Search"
+  }
+
+  # search_in_session_index: number
+  # Table reference - INT8 ENCODE ZSTD
+  #
+  # This dimension is used in calculated fields that are more meaningful for Explores (max_search_index);
+  #  on it's own, search_in_session_index it is difficult to describe succinctly, and may be more confusing
+  #  than helpful to Looker users developing Explores, so it has been hidden.
+  dimension: search_in_session_index {
+    description: "The search index within a single session."
+    type: number
+    # index within each session
+    sql: ${TABLE}.search_in_session_index ;;
+    group_label: "Search"
+    hidden: yes
+  }
+
+  # TODO: flag for removal; this is a URI query component that is better represented to users by readable_terms.
+  dimension: search_field {
+    type: string
+    # sql: decode(split_part(${page_url},'/search/',2),'%20', ' ');;
+    sql: REPLACE(split_part(${page_url},'/search/',2), '%20', ' ')
+      ;;
+    hidden: yes
+  }
+
   # Search results
 
+  # terms: string
+  #
+  # Table reference - VARCHAR(2048) ENCODE RAW
   # terms returns a string array of search terms
-  # which can have a size > 1
+  # The array can have a size greater than 1 in multi-dimension searches
+  # TODO: GDXDSD-1326
   dimension: terms {
     type: string
     sql: ${TABLE}.terms ;;
     group_label: "Results"
   }
 
-  # readable_terms removes wrapping [" "] from the terms dimension,
-  # array and replaces + signs with ' ', rendering a readable string
-  # in the case of the term string array containing a single element
+  # readable_terms: string
+  #
+  # removes wrapping [" "] from the terms dimension, array and
+  # replaces + signs with ' ', rendering a readable string in
+  # the case of the term string array containing a single element
+  # TODO: GDXDSD-1326 handle multiple-term searches past the first index, and empty searches.
   dimension: readable_terms {
```

Comment:
Should this have a description? People my ask what are readable terms

Reply:
Yes. I've added `description: "The search term(s) that were queried."` and will be renaming the dimension as `search_terms`. I will also rename the "unprocessed" terms dimension to as `raw_search_terms` and hide it since it is not the preferred dimension for users.